src/EditorFeatures/CSharpTest/CodeActions/LambdaSimplifier/LambdaSimplifierTests.cs

```diff
@@ -344,7 +344,7 @@ static void Bar(Action<string> x)
 
     static void Main()
     {
-        Bar(x => [||]Goo(x));
+        Bar(x [||]=> Goo(x));
```

Comment:
fwiw, in general it would be preferable to keep the orignal test around, just with the new expected result (in this case 'missing').  Then add a new test for this new scenario.

Reply:
There's a new explicit test for missing in that case `TestMissingCaretPositionBeforeBody`. This test was testing something else and thus had to be fixed.